clients/banking/src/utils/templateTranslations.ts

```diff
@@ -40,3 +40,8 @@ export const getInternationalTransferFormRouteLabel = (route: string) => {
     .with(P.when(isTranslationKey), key => t(key))
     .otherwise(() => route);
 };
+
+export const formatPascalCaseToWords = (text: string): string =>
```

Comment:
if we still keep it despite my previous comment, I think we should add some unit test